packages/swayze/lib/src/config.dart

```diff
@@ -24,5 +24,7 @@ double headerWidthForRange(Range range) {
 const kDefaultCellWidth = 120.0;
 const kDefaultCellHeight = 33.0;
 
+const kMinCellWidth = 30.0;
```

Comment:
This should, probably, be configurable by the user. This can be merged as is, but an US should be created to allow the user to override these constants (not just this particular one)